evennia/web/webclient/static/webclient/css/webclient.css

```diff
@@ -94,31 +94,54 @@ div {margin:0px;}
 
 /* Main scrolling message area */
 #messagewindow {
-    position: relative;
-    height: 90%;
+    position: absolute;
     overflow: auto;
     padding: 1em;
+    box-sizing: border-box;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 70px;
 }
 
 /* Input area containing input field and button */
 #inputform {
     position: absolute;
     width: 100%;
-    padding: .8em 0;
-    bottom: -50px;
+    padding: 0;
+    bottom: 0;
+    margin: 0;
+    padding-bottom: 10px;
+    border-top: 1px solid #555;
 }
 
 #inputcontrol {
-    padding: 0 .8em;
-    overflow: auto
+    width: 100%;
+    padding: 0;
 }
 
 /* Input field */
+#inputfield, #inputsend, #inputsizer {
+    display: block;
+    box-sizing: border-box;
```

Comment:
Ah yes, I'll add those. I'm used to the CSS preprocessor I use at work, which adds them automatically. But yeah, the vast majority of browsers can handle it these days, and even IE8 had support for it. There's a scary-looking warning on it for MDN, but in practice I see it used quite a bit.